lib/pages/settings_dashboard/settings/settings.dart

```diff
@@ -64,7 +68,7 @@ class SettingsController extends State<Settings> with ConnectPageMixin {
       settingEnum == optionsSelectNotifier.value;
 
   void logoutAction() async {
-    final noBackup = showChatBackupBanner == true;
+    final noBackup = showChatBackupSwitch.value == true;
```

Comment:
why put backup in `logout` action?

Reply:
I just changed `showChatBackupSwitch` and kept old logic.
IMO: It's used for UI